schemas/dto/rawRecordsMetadata.json

```diff
@@ -0,0 +1,35 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Schema that describes metadata for receiving a chunk of raw records",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "last": {
+      "description": "Indicates if the current chunk is the last one associated with the same JobExecution",
+      "type": "boolean"
+    },
+    "counter": {
+      "description": "Counter of records associated with the same JobExecution, the last counter number is a total number of records",
+      "type": "integer"
+    },
+    "total": {
+      "description": "Total number of records associated with the same JobExecution",
+      "type": "integer"
+    },
+    "contentType": {
+      "description": "Describes type of records and format of record representation",
+      "type": "string",
+      "enum": [
+        "MARC_RAW",
+        "MARC_JSON",
+        "MARC_XML"
+      ]
+    }
+  },
+  "required": [
+    "last",
+    "counter",
+    "records",
```

Comment:
I don't see "records" field here, did you mean "total"?

Reply:
total is not req field, but yes. Should delete records